src/main/java/bio/terra/workspace/service/resource/controlled/AccessScopeType.java

```diff
@@ -7,41 +7,38 @@
 
 /** Controlled resources can be shared by all users in a workspace or be private to a single user */
 public enum AccessScopeType {
-    ACCESS_SCOPE_SHARED("ACCESS_SHARED"),
-    ACCESS_SCOPE_PRIVATE("ACCESS_PRIVATE");
-
-    private final String dbString;
-
-    AccessScopeType(String dbString) {
-        this.dbString = dbString;
-    }
-
-    public String toSql() {
-        return dbString;
+  ACCESS_SCOPE_SHARED("ACCESS_SHARED"),
+  ACCESS_SCOPE_PRIVATE("ACCESS_PRIVATE");
+
+  private final String dbString;
+
+  AccessScopeType(String dbString) {
+    this.dbString = dbString;
+  }
+
+  public String toSql() {
+    return dbString;
+  }
+
+  public static AccessScopeType fromApi(
+      ControlledResourceCommonFields.AccessScopeEnum apiAccessScope) {
+    switch (apiAccessScope) {
+      case PRIVATE_ACCESS:
+        return ACCESS_SCOPE_PRIVATE;
+      case SHARED_ACCESS:
+        return ACCESS_SCOPE_SHARED;
+      default:
+        throw new InternalLogicException("Unknown API access scope");
     }
+  }
 
-    public static AccessScopeType fromApi(
-            ControlledResourceCommonFields.AccessScopeEnum apiAccessScope
-    ) {
-        switch (apiAccessScope) {
-            case PRIVATE_ACCESS:
-                return ACCESS_SCOPE_PRIVATE;
-            case SHARED_ACCESS:
-                return ACCESS_SCOPE_SHARED;
-            default:
-                throw new InternalLogicException("Unknown API access scope");
-        }
-
-    }
-
-    public static AccessScopeType fromSql(String dbString) {
-        for (AccessScopeType value : values()) {
-            if (StringUtils.equals(value.dbString, dbString)) {
-                return value;
-            }
-        }
-        throw new SerializationException(
-                "Deeserialization failed: no matching access scope type for " + dbString);
+  public static AccessScopeType fromSql(String dbString) {
+    for (AccessScopeType value : values()) {
```

Comment:
We can write a generic method that does this for any enum and call that.

Reply:
I tried writing that. It is quite messy. Maybe you can do a better job.
I'm going to leave this in the interest of getting it done.
There is a shorter expression using streams like:
```
Arrays.stream(values())
            .filter(e -> StringUtils.equals(e, dbString)
            .findFirst()
            .orElseThrow(() -> SerializationException(...));
```